baseplate/sidecars/live_data_watcher.py

```diff
@@ -1,18 +1,30 @@
 """Watch nodes in ZooKeeper and sync their contents to disk on change."""
+from gevent.monkey import patch_all
```

Comment:
```suggestion
# pylint: disable=wrong-import-position,wrong-import-order
from gevent.monkey import patch_all
```

I think this should fix the linting issue

Reply:
Added - thanks, fighting nixos to try to get the linting setup locally...

Reply:
one of the docs in the repo suggests how to do that using docker-compose :)

Reply:
fixed, thanks - used docker compose :+1: